src/Aspire.Hosting/Dashboard/TransportOptions.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Dashboard;
+
+internal class TransportOptions
+{
+    public bool? AllowUnsecureTransport { get; set; }
```

Comment:
Do we need to differentiate null vs false anywhere?

Reply:
Nope. I've changed this to just a plain old bool. We resolve the truthiness in PostConfigure so we know what this value is by the time people are reading this.